src/App.js

```diff
@@ -0,0 +1,31 @@
+import React from 'react';
+import './assets/main.css';
+import { Routes, Route } from 'react-router-dom';
+import Calculator from './components/Calculator';
+import Home from './components/Home';
+import Quote from './components/Quote';
+import Navbar from './components/Navbar';
+
+class App extends React.Component {
+  constructor(props) {
+    super(props);
+    this.state = {};
+  }
+
+  render() {
+    return (
+      <div className="App">
+        <section className="app-container">
+          <Navbar />
+          <Routes>
+            <Route path="/" element={<Home />} />
+            <Route path="/calculator" element={<Calculator />} />
+            <Route path="/quote" element={<Quote />} />
+          </Routes>
+        </section>
+      </div>
+    );
+  }
+}
+
```

Comment:
- [ ] Please your app is not working. Kindly check and find out where the mistake is coming from.
- 

![2](https://user-images.githubusercontent.com/63972670/173183827-d015c0fe-8a8a-49f8-94f1-7e6b4ddf6d40.PNG)
- Suggestions: Check the spelling of `Navbar` and `Calculator`. Check if small and capital letters match with `components`